.github/workflows/ci_e2e_workflow.yml

```diff
@@ -38,6 +38,10 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '6.x'
```

Comment:
If this is not used then lets remove the old dotnet versions

Reply:
not too certain, will address in next pr